### DISCUSSION TITLE
Alpyne Stock Management Game Trained RL Model Example

### DISCUSSION THREAD

**Discussion by rokitatomer** [the-anylogic-company/Alpyne#13] [Q&A]
Hello All,
I am trying to play and learn Alpyne. I have started with the Stock example. I have trained a model using the rlpolicy_train.py script , and got an output model StockGamePolicy_PPO.zip. I am trying to use this model in order to predict the best order rate, however I am getting very bad results. I am attaching the trained model and the scripts that load the model.
I was wondering if anyone got a good model and could share their trained model and a script the utilize this model - I guess I am doing something wrong.

Best,
Tomer
[StockGamePolicy_PPO.zip](https://github.com/t-wolfeadam/Alpyne/files/7965387/StockGamePolicy_PPO.zip)
[simple run - multiRL.txt](https://github.com/t-wolfeadam/Alpyne/files/7965389/simple.run.-.multiRL.txt)

**Comment by negodfre** [comment]
When I use the "check_env" function, it mentions action space should be symmetric and normalized between [-1, 1].
I wonder if it's having difficulty optimizing without the normalized action space.

It references this documentation:
https://stable-baselines3.readthedocs.io/en/master/guide/rl_tips.html

The import for check_env would be the following:
`from stable_baselines3.common.env_checker import check_env`

I am actually having difficulty getting the program to run using make_vec_env().
If I try running the environment with one CPU, it looks like it's going to take 16 hours.
